evap/evaluation/management/commands/anonymize.py

```diff
@@ -205,8 +216,36 @@ def anonymize_questionnaires(self, lorem_ipsum):
                 text_answer.original_answer = self.lorem(text_answer.original_answer, lorem_ipsum)
             text_answer.save()
 
-    # Returns a string with the same number of words as the given text, but
-    # uses lorem ipsum instead of the actual words.
+        self.stdout.write("Shuffling answer counters...")
+        for contribution in Contribution.objects.all():
+            for questionnaire in contribution.questionnaires.all():
+                for question in questionnaire.questions.all():
+                    rating_answer_counters = RatingAnswerCounter.objects.filter(
+                        contribution=contribution,
+                        question=question
+                    )
+                    answer_count = sum(
+                        [rating_answer_counter.count for rating_answer_counter in rating_answer_counters]
+                    )
+                    answers = [rating_answer_counter.answer for rating_answer_counter in rating_answer_counters]
+                    for answer in answers:
+                        counter = RatingAnswerCounter.objects.get(
+                            contribution=contribution,
+                            question=question,
+                            answer=answer
+                        )
+                        if answer == answers[-1]:
+                            new_count = answer_count
```

Comment:
why?

Reply:
the last option gets all "remaining" answers that have not been assigned to other options.